### PR TITLE
Document the user workflow

### PR DESCRIPTION
**Pull request by eccles**
Problem:
Access to this repo must be available to any user on github.

Solution:
Describe in README the acceptable workflow using the 'fork' option in
the github UI.

Signed-off-by: eccles <phewlett76@gmail.com>

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+*  @JAG-UK @j-hartley @serhiy1 @leflambeur
```
Comment (jitsuineccles) [inline .github/CODEOWNERS:1]: I dont think this needed just yet. Anyway this file must be added in its own commit - please delete

**Review by jitsuineccles — CHANGES_REQUESTED** [review]

**Review by jitsuineccles — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,18 +1,51 @@
...
+### Preparation
+
+Reference: https://gist.github.com/Chaser324/ce0505fbed06b947d962
+
+Fork the repo using the 'Fork' dialog at the top right corner of the github UI.
+
+and add the original upstream repo as a remote:
+
+```bash
+git remote add upstream git@github.com:jitsuin-inc/archivist-shell.git
```
Comment (jitsuineccles) [inline README.md:25]: This should go after the clone step surely?

**Review by jitsuineccles — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jitsuineccles) [reply README.md]: fixed

**Review by jitsuineccles — APPROVED** [review]

**Review by leflambeur — CHANGES_REQUESTED** [review]
Minor fixes - we need to Americanize things like synchronizing instead of synchronising

Worth doing a re-check of this and archivist-python to make sure it matches the docs style guide - link for ref. https://jitsuin-inc.github.io/rkvst-docs/contributing/formatting-content/style-guide/#grammar

Once you have reviewed it and updated those please squish the commits and I will approve it and merge

**Review by leflambeur — APPROVED** [review]

**Comment by leflambeur** [comment]
LGTM and all checks passed, will merge now

**Pull request merged**